mods/taskbar-toolbar-separators.wh.cpp

```diff
@@ -0,0 +1,109 @@
+// ==WindhawkMod==
+// @id              taskbar-toolbar-separators
+// @name            Taskbar Toolbar Separators
+// @description     Enables seperators between taskbar toolbars
+// @version         1.0.0
+// @author          ItsProfessional
+// @github          https://github.com/ItsProfessional
+// @include         explorer.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Taskbar Toolbar Separators
+This mod enables seperators between taskbar toolbars.
+
+**NOTE: YOU HAVE TO RESTART YOUR COMPUTER FOR THE MOD TO TAKE EFFECT!**
+
+**Without** this mod:
+![Screenshot](https://i.imgur.com/hcNVQgv.png)
+
+**With** this mod:
+![Screenshot](https://i.imgur.com/h2IfdOg.png)
+*/
+// ==/WindhawkModReadme==
+
+#include "commctrl.h"
+
+using CreateWindowExW_t = decltype(&CreateWindowExW);
+CreateWindowExW_t pOriginalCreateWindowExW;
+HWND CreateWindowExWHook(
+    DWORD     dwExStyle,
+    LPCWSTR   lpClassName,
+    LPCWSTR   lpWindowName,
+    DWORD     dwStyle,
+    int       X,
+    int       Y,
+    int       nWidth,
+    int       nHeight,
+    HWND      hWndParent,
+    HMENU     hMenu,
+    HINSTANCE hInstance,
+    LPVOID    lpParam
+)
+{
+    if ((*((WORD*)&(lpClassName)+1)) && !wcscmp(lpClassName, L"ReBarWindow32"))
```

Comment:
```
*((WORD*)&(lpClassName)+1)
```
That's not a correct check since it only checks 2 bytes out of 8. The pointer can be, for example, `0x0000_1234_0000_0000` and your check will skip it.